www/index.html

```diff
@@ -38,23 +20,154 @@
     -->
     <!-- <meta http-equiv="Content-Security-Policy" content="default-src 'self' data: gap: 'unsafe-inline' https://ssl.gstatic.com; style-src 'self' 'unsafe-inline'; media-src *" /> -->
 
-    <link rel="stylesheet" type="text/css" href="css/index.css" />
-    <title>Hello World</title>
+    <link rel="stylesheet" href="./libs/bootstrap/3.3.7/css/bootstrap.min.css" >
```

Comment:
I chose Bootstrap as CSS framework becasue:
1. It is a mobile first CSS framework,
2. I have experience working with it,
3. Because It has a large community and a documentation I'm familiar with. 

I don't have vast knowledge on CSS frameworks, thus I could not compare it.